requirements-dev.txt

```diff
@@ -4,9 +4,7 @@
 #
 #    make up
 #
-aenum==2.1.2              # via pypika
-aiocontextvars==0.2.2 ; python_version < "3.7"
-aiofiles==0.4.0           # via quart
+aiofiles==0.4.0           # via quart, sanic
```

Comment:
This is related to a fix I did in this commit: https://github.com/tortoise/tortoise-orm/pull/162/commits/b4a9ce7f578b431374221fb08d6fa868b40833a6#diff-b67911656ef5d18c4ae36cb6741b7965
That should fix requirement generation to not assume that your current environment is what this is going to run on
(We have CI on Python3.5 & Python3.6).
In some ways I'd suggest reverting the requirements*.txt files, and manually addiing `sanic;python_version>="3.6"` to `requirements-dev.txt` for now. Then this will get properly fixed in after #162 gets merged.

(I would prefer to merge yours first, then all the small changes I did is my responsibility to ensure they work)